deploy/deploy.js

```diff
@@ -6,8 +6,8 @@ module.exports = async ({ deployments, getNamedAccounts }) => {
     console.log('network id ', await getChainId());
 
     const {deploy} = deployments;
-    const {deployer, tokenContract, rewardsHolder} = await getNamedAccounts();
-    const args = [tokenContract, rewardsHolder];
+    const {deployer, tokenContract, rewardsHolder, owner} = await getNamedAccounts();
+    const args = [tokenContract, rewardsHolder, owner];
```

Comment:
If we merge #18, the arguments of the constructor must include TokenStaking contract address.

Reply:
Finally #18 won't be merged

https://github.com/threshold-network/merkle-distribution/pull/18#issuecomment-1184720318